src/Symfony/Component/Debug/Tests/Fixtures/FinalClasses.php

```diff
@@ -64,7 +64,6 @@ class FinalClass6
  * @author John Doe
  *
  * @final another
- *
```

Comment:
this change had to be reverted as otherwise the second line is not part of the multiline comment

Reply:
I'm sorry but isn't it a BC to alter a test case? Those cases were made on purpose. If they don't fit SF CS standard, we better exclude them from the analysis.

Reply:
BC breaks relate to the interface, here it's a behavior change :) - considered a bug fix.
If you don't agree, can you find some kind of "proof" that blank lines in multi line comments are "legal"?

Reply:
I'm think of the case where you use the component outside of SF framework ofc.

Reply:
Sorry @nicolas-grekas, wrote my last message without seeing yours. I don't have any proof. Just that if we can support more cases, why wouldn't we do it? I tried to support as many cases as possible when I did this so let's keep it that way and ignore those fixtures files in CS checks instead?

Reply:
It's not related to Symfony. If phpdocs do not support a blank line in multiline comments, we must not support it either.

Reply:
The problem with PHPDoc is that it's not clear at all on edge cases... Since I didn't find an example on the website that have a multiline comment with a blank line, I guess you are right. We can keep it like this 😄

Reply:
Thank you for the feedback @fancyweb!